start/data/file-transfer-via-http.md

```diff
@@ -0,0 +1,43 @@
+
+[title]: - "File Transfer via HTCondor"
+ 
+
+# File Transfer via HTCondor
+
+## Overview
+
+Due to the distributed configuration of the OSG, more often than not, your jobs will need to bring along a copy 
+(i.e. transfer a copy) of data, code, packages, software, etc. to the execute node where the job will run. 
+This requirement applies to any and all files that are needed to successfully execute and complete your job.
+
+This guide will describe steps and important considerations for transferring your files that are >100MB but <1GB 
+using the HTTP file transfer mechanism via the HTCondor submit file.
+
+## Important Considerations
+
+As described in the [Introduction to Data Management on OSG Connect](https://support.opensciencegrid.org/support/solutions/articles/12000002985) 
+any data, files, or even software that is >100MB should be staged in your `/public` directory on your login node. Files in your 
+`/public` directory that are <1GB can be transferred with your jobs via HTTP using your HTCondor submit file.
+
+If you have files that are >1GB please plan to use [StashCache](https://support.opensciencegrid.org/support/solutions/articles/12000002775).
+
+## Transfer Files From `/public` Using HTTP
+
+To transfer files staged in your `/public` directory, use an HTTP URL in combination with the `transfer_input_files` 
+statement in your HTCondor submit. For example:
+
+	transfer_input_files = http://stash.osgconnect.net/public/username/path/my_software.tar.gz
+
+Where `username` refers to your OSG Connect username. Multiple URLs can be specified using a comma-separated list, and 
+a combination of URLs and files from `/home` directory can be provided in a comma separated list. For example,
+
+	transfer_input_files = http://stash.osgconnect.net/public/username/path/my_software.tar.gz, /home/username/my_data.csv
+

```

Comment:
Maybe add something about how "my_software.tar.gz" would be 100Mb - 1Gb, my_data.csv would be smaller. Not necessary, just a thought...